quickstarts/gradle/micronaut4/README.md

```diff
@@ -0,0 +1,46 @@
+---
+name: "Gradle :: Micronaut 4"
+description: Micronaut 4 application featuring REST endpoints (micronaut-http) with validation (micronaut-validation).
+---
+# Eclipse JKube Micronaut 4 quickstart
+
+Micronaut 4 application featuring REST endpoints (micronaut-http) with validation (micronaut-validation).
+
+## Requirements
+
+- JDK 17+
+- Kubernetes Cluster (Minikube, OpenShift, CRC, etc.)
+
+## Building and deploying the application
+
+```shell script
+# Kubernetes
+$ gradle clean build k8sBuild k8sResource k8sApply
+# OpenShift
+$ gradle clean build ocBuild ocResource ocApply
+```
+## Expected output
+
+Once you've deployed the application, you should be able to perform requests:
+
+### Minikube
+
+```shell script
+$ curl $(minikube ip):$(kubectl get svc micronaut-zero-config -n default -o jsonpath='{.spec.ports[].nodePort}')
+Hello from Micronaut deployed with JKube!
+$ curl $(minikube ip):$(kubectl get svc micronaut-zero-config -n default -o jsonpath='{.spec.ports[].nodePort}')/hello/world
+Hello word!
+$ curl $(minikube ip):$(kubectl get svc micronaut-zero-config -n default -o jsonpath='{.spec.ports[].nodePort}')/hello/1
+{"message":"name: size must be between 3 and 2147483647","_links":{"self":{"href":"/hello/1","templated":false}}}
```

Comment:
Is this response expected?

Reply:
Document was updated to reflect changes in source code.
Deployment was test on both k8s and ocp (rh sandbox) - it is working.

Reply:
Document was updated to reflect changes in source code.
Deployment was test on both k8s and ocp (rh sandbox) - it is working.